h/static/scripts/annotator/guest.coffee

```diff
@@ -384,11 +411,10 @@ module.exports = class Guest extends Annotator
     event.preventDefault?()
     event.stopPropagation?()
     @adder.hide()
-    switch event.target.dataset.action
```

Comment:
This doesn't seem to change any behavior. Maybe doesn't need to be included here.